packages/taquito-michel-codec/src/michelson-types.ts

```diff
@@ -386,6 +386,7 @@ export enum Protocol {
   Psithaca2 = 'Psithaca2MLRFYargivpo7YvUr7wUDqyxrdhC5CQq78mRvimz6A',
   PtJakarta = 'PtJakartaiDz69SfDDLXJSiuZqTSeSKRDbKVZC8MNzJnvRjvnGw',
   PtJakart2 = 'PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY',
+  PtKathman = 'PtKathmanYPEsBau83zRwXK19HAQRzZSj6pCXnvtyLZFjRxdQxt',
```

Comment:
It should be PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg.

Reply:
fixed